hack/sync.sh

```diff
@@ -16,6 +16,30 @@ fi
 
 source $KUBEVIRTCI_PATH/hack/common.sh
 
+OLM_NS=openshift-marketplace
+TARGET_NS=openshift-cnv
+
+# Deploy OLM if needed
+if [[ $KUBEVIRT_PROVIDER = k8s* ]]; then
+  SELF=$( realpath $0 )
+  BASEPATH=$( dirname $SELF )
+  . "${BASEPATH}/gen-operator-sdk.sh"
+
+  set +e
+  ${OPERATOR_SDK} olm status
+  if [[ $? != 0 ]] ; then
+    ${OPERATOR_SDK} olm install
```

Comment:
Wasn't there a whole script somewhere that was trying to do the same as this one line?

Reply:
I don't think we installed OLM earlier, so no old script.